hyperwallet/utils/apiclient.py

```diff
@@ -112,6 +112,13 @@ def _makeRequest(self,
 
         if response.status_code is 204:
             return {}
+        elif response.status_code is 429:
```

Comment:
@m-aciek I agree with your implementation, but I just want to ask you to provide the tests to cover it at `test_client.py`

Reply:
@wmews-hw I've added a test in `test_client.py`